backend/src/controllers/teamController.ts

```diff
@@ -61,6 +62,14 @@ const addTeam = async (req: Request, res: Response) => {
       },
     });
 
+    addNotification(
+      members.map((member: User) => member.address),
+      {
+        message: `You have been added to a new team: ${name} by ${owner.name}`,
+        subject: "Team Invitation",
+        author: owner.name?.toString() || "",
```

Comment:
Author should be the author address, so you can query notifications based on that. Should've been more descriptive